docs/how-to/profiling.txt

```diff
@@ -0,0 +1,81 @@
+.. _howto-profiling:
+
+#################################
+ How To: Profile Your Experiment
+#################################
+
+When you want to improve the training speed of your model, the first
+thing you need to do is understand why training is slow. To do this, you
+need to profile your training code. There are many different layers you
+can look at, from raw throughput all the way to GPU kernals.
+
+Determined provides two tools out-of-the-box for profiling your training
+- System Metrics and Timings.
+
+****************
+ System Metrics
+****************
+
+System Metrics are statistics around hardware usage, such as GPU
+utilization and network throughput. These metrics are useful for seeing
+whether your training is using the hardware effectively. When the System
+Metrics reported for an experiment are below what you expect from your
+hardware, that is a sign that the software may be able to be optimized
+to make better use of the hardware resources.
+
+Specifically, Determined tracks:
+
+-  GPU utilization
+-  GPU memory usage
+-  Network throughput (sent)
+-  Network throughput (received)
+-  Disk IOPS
+-  Disk throughput (read)
+-  Disk throughput (write)
+-  Host memory sage
+-  Average CPU utilization across cores
+
+For distributed training, these metrics are collected for every agent.
+The data is broken down per agent. GPU metrics can be further broken
+down by GPU.
+
+Note: System Metrics record agent-level metrics, so when there are
+multiple experiments on the agent, it is difficult to analyze. We
+suggest that profiling is done when the experiment is the only thing
+running on the agent.
+
+*******************************
```

Comment:
I could take or leave this, or add it to a follow up if you wanted (seems it this could be a topic guide on profiling in determined and the other (this) could be a how to?).

Reply:
I removed it. It can be an add on-later